src/AppBundle/Controller/Admin/VoteSettingsController.php

```diff
@@ -0,0 +1,246 @@
+<?php
+
+namespace AppBundle\Controller\Admin;
+
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use AppBundle\Entity\VoteSettings;
+use AppBundle\Form\VoteSettingsType;
+
+/**
+ * VoteSettings controller.
+ *
+ * @Route("/admin/vote_settings")
+ */
+class VoteSettingsController extends Controller
+{
+    /**
+     * Lists all VoteSettings entities.
+     *
+     * @Route("/", name="vote_settings")
+     * @Method("GET")
+     * @Template()
+     */
+    public function indexAction()
+    {
+        $em = $this->getDoctrine()->getManager();
+
+        $entities = $em->getRepository('AppBundle:VoteSettings')->findAll();
+
+        return array(
+            'entities' => $entities,
+        );
+    }
+    /**
+     * Creates a new VoteSettings entity.
+     *
+     * @Route("/", name="vote_settings_create")
+     * @Method("POST")
```

Comment:
You allow to enter in this action only from the method POST, but how can this action made form view(in line 60) and pass it to template? This action must have and GET method also. That's why  you have to change name of route to "new" or "create", for example. If your method will have creating form view (and pass it into template) and parse form by POST/PUT/DELETE Methods in one action it is more readable.